packages/orchestrator-ui-components/src/components/WFOButtonComboBox/WFOButtonComboBox.tsx

```diff
@@ -14,29 +16,38 @@ export type WFOButtonComboBoxProps = {
     buttonText: string;
     options: WorkflowComboBoxOption[];
     onOptionChange: (selectedOption: WorkflowComboBoxOption) => void;
+    isProcess: boolean;
```

Comment:
If you change this into `isProcess?: boolean` and set the default value to false, you can omit the prop when it is not a process.

Reply:
I know -> I made it this explicit by design.

There will be more places that do stuff for Tasks Or Processes; this way you can always know which one.

I am open for a better name though. We now mix, Task, Process, Workflows and Products in unreliable/inconsistent ways: Consider the `WFOStartCreateWorkflowButton` vs `WFOStartTaskButton` -> They both start a Process...